scib/preprocessing.py

```diff
@@ -264,18 +264,26 @@ def scale_batch(adata, batch):
     return adata_scaled
 
 
-def hvg_intersect(adata, batch, target_genes=2000, flavor='cell_ranger', n_bins=20, adataOut=False, n_stop=8000,
-                  min_genes=500, step_size=1000):
-    ### Feature Selection
+def hvg_intersect(
+        adata,
+        batch,
+        target_genes=2000,
+        flavor='cell_ranger',
+        n_bins=20,
+        adataOut=False,
+        n_stop=8000,
+        min_genes=500,
+        step_size=1000
+):
```

Comment:
I think the scanpy function now does batch HVG selection. Could you use that instead?

Reply:
I think we discussed that at some point but ultimately decided on our own implementation. @LuckyMD would this be something to reconsider?

Reply:
When we decided to implement this, the scanpy function didn't yet exist... then goekcen wrote it before we were done. It should be more or less the same. For consistency I think we should keep this. Also because I have other code in a reproducibility github somewhere that uses it.

Reply:
Keeping this for now